e2e-tests/.ci/server.generate.sh

```diff
@@ -78,7 +78,7 @@ $(for service in $ENABLED_DOCKER_SERVICES; do
 $(if mme2e_is_token_in_list "postgres" "$ENABLED_DOCKER_SERVICES"; then
     echo '
   postgres:
-    image: mattermostdevelopment/mirrored-postgres:12
+    image: mattermostdevelopment/mirrored-postgres:13
```

Comment:
E2E failed since this image is not yet available. Currently, the mirroring will happen on merge to master. Probably need to separate the changes to `server/scripts/mirror-docker-images.json` and merge first, we can update/run E2E here and see if nothing breaks. cc @mvitale1989

Reply:
That's right, thanks Saturn. Here is the PR: https://github.com/mattermost/mattermost/pull/30047
I have already run the mirroring script locally and pushed the postgres:13 image, so we can already run the tests in this PR